Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,16 +1,39 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.IO;
-using System.Linq;
-using System.Text;
+﻿using System.Text;
+using static BootCamp.Chapter.TextTable;
+using static System.Console;
 
 namespace BootCamp.Chapter
 {
-    public class Program
+    public static class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
+            Demo();
+        }
+
+        private static void Demo()
+        {
+            // set currency symbol and position for BlanceStats.FormatCurrency method
+            BalanceStats.DefaultCurrencySymbol = "€";
+            BalanceStats.DefaultCurrencySymbolPosition = 1;
+
+            const int padding = 3;
+            OutputEncoding = Encoding.UTF8;
+
+            // runs statistics methods on Ballances.corrupted
+
+            var repairedPeopeAndBalances = FilesUtility.MakeBalancesParsable();
+
+            WriteLine(Build(BalanceStats.FindHighestBalanceEver(repairedPeopeAndBalances), padding));
+
+            WriteLine(Build(BalanceStats.FindMostPoorPerson(repairedPeopeAndBalances), padding));
+
+            WriteLine(Build(BalanceStats.FindPersonWithBiggestLoss(repairedPeopeAndBalances), padding));
+
+            WriteLine(Build(BalanceStats.FindRichestPerson(repairedPeopeAndBalances), padding));
+
+            // Creates Balances.repaired
```

Comment:
The code speaks for itself. You don';t have to write a comment, in fact by doing so you make readability worse with noise.